src/regenerate.rs

```diff
@@ -120,7 +122,7 @@ fn get_corresponding_type(elements: &[(Option<TypeStruct>, Vec<String>)],
 // The hashmap key is `Some(file name)` or `None` for entries that ignore file name
 pub fn regenerate_comments(work_dir: &Path, path: &str,
         infos: &mut HashMap<Option<String>, Vec<(Option<TypeStruct>, Vec<String>)>>,
-        ignore_macros: bool) {
+        ignore_macros: bool, ignore_doc_commented: bool) {
```

Comment:
The code remains readable for the moment. But yes, if we add another option, I'll use a bitflag instead.